rmf_fleet_adapter_python/scripts/rmf_fleet_adapter_python/test_adapter.py

```diff
@@ -22,13 +25,21 @@
 pickup_name = "pickup"
 dropoff_name = "dropoff"
 
-quiet_dispenser_name = "quiet"
-flaky_dispenser_name = "flaky"
+dispenser_name = "mock_dispenser"
+ingestor_name = "mock_ingestor"
 
 ###############################################################################
 # CONSTS
 ###############################################################################
 
+INGESTOR_RESULT_ACKNOWLEDGED = 0
+INGESTOR_RESULT_SUCCESS = 1
+INGESTOR_RESULT_FAILED = 2
+
+INGESTOR_STATE_IDLE = 0
+INGESTOR_STATE_BUSY = 1
+INGESTOR_STATE_OFFLINE = 2
```

Comment:
Nitpick here. Instead of declaring the macro here, would prefer to directly use the enum in `rmf_dispenser_msgs` and `rmf_ingestor_msgs`.

Reply:
done!